atlasdb-config/src/main/java/com/palantir/atlasdb/factory/Leaders.java

```diff
@@ -62,22 +62,22 @@ private Leaders() {
      * Creates a LeaderElectionService using the supplied configuration and
      * registers appropriate endpoints for that service.
      */
-    public static LeaderElectionService create(Environment env, LeaderConfig config) {
```

Comment:
any reason to change this to a `Consumer`? it's a little less obvious what it is now

Reply:
(wouldn't be opposed to renaming the class `EndpointRegistrar`

Reply:
Mostly it just seemed unnecessary? the interface is exactly that of a Consumer. And I've had issues in the past where I needed to pull atlasdb-config sadness on to my classpath just to get access to that interface (though on 2nd thought I could have worked around that).

Reply:
Also makes it match how TimelockAgent works